zhaquirks/innr/rb285c.py

```diff
@@ -0,0 +1,82 @@
+"""Innr RB 285 C device."""
+from zigpy.profiles import zha
+from zigpy.profiles.zha import DeviceType
+from zigpy.quirks import CustomDevice
+from zigpy.zcl.clusters.general import (
+    Basic, GreenPowerProxy, Groups, Identify, LevelControl, OnOff, Ota, Scenes)
+from zigpy.zcl.clusters.lighting import Color
+from zigpy.zcl.clusters.lightlink import LightLink
+
+
+class RB285C(CustomDevice):
+    """Innr RB 285 C device."""
+
+    signature = {
+        # <SimpleDescriptor endpoint=1 profile=260 device_type=269
+        # device_version=1
+        # input_clusters=[0, 3, 4, 5, 6, 8, 768, 4096]
+        # output_clusters=[25]>
+        1: {
+            'manufacturer': 'innr',
+            'model': 'RB 285 C',
+            'profile_id': zha.PROFILE_ID,
+            'device_type': 269,
```

Comment:
Please use a constant for this.

Reply:
Sorry, should've read the contributer documentation about this, just copied the existing innr quirk. 

I found out that this device type is an 'Extended Colour Light', so probably we need to add support to home assistant/zigpy for these types so we don't need quirks. What do you think?

Reply:
The device types are defined in Zigpy here: https://github.com/zigpy/zigpy/blob/a607f6284cd9dffbe7030ba265f9c7d3a02d95d1/zigpy/profiles/zha.py#L7 and here: https://github.com/zigpy/zigpy/blob/a607f6284cd9dffbe7030ba265f9c7d3a02d95d1/zigpy/profiles/zll.py#L7

and then that info is linked to domains in ZHA/HA here: 
https://github.com/home-assistant/home-assistant/blob/b71baef7c8effa036c17e4955bdf829580b1401a/homeassistant/components/zha/core/registries.py#L112

so you could PR the changes there instead and avoid the quirks all together.